allow-trunk-api-policy

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Is this supposed to be in the update version commit?

Reply:
No this should not be. Will fix.